internal/cmd/run_test.go

```diff
@@ -95,7 +97,7 @@ func TestGetEnvoyRun_ReadsHomeVersionFile(t *testing.T) {
 	runWithoutConfig(t, c)
 
 	// No implicit lookup
-	require.NotContains(t, o.Out.(*bytes.Buffer).String(), "looking up latest version\n")
+	require.NotContains(t, o.Out.(*bytes.Buffer).String(), "looking up latest version"+lf)
```

Comment:
I guess we could use `fmt.Sprintln` to produce a portable string with a trailing new line..

Reply:
except it doesn't :) inside it does this

```
	p.buf.writeByte('\n')
```

Reply:
it is very strange that go decided to not include portable linefeed support 🤷

Reply:
funny 😂

Reply:
that said, we could make our own Sprintln that does..

Reply:
meh... problem with doing this is that IDEs won't syntax highlight the format string